Gems/Atom/Feature/Common/Assets/Shaders/Materials/DepthPass_WithPS.azsli

```diff
@@ -92,16 +91,25 @@ struct PSDepthOutput
 
 PSDepthOutput MainPS(VSDepthOutput IN, bool isFrontFace : SV_IsFrontFace)
 {
-    const float3 vertexNormal = normalize(IN.m_normal);
+    float3 vertexNormal = IN.m_normal;
+    float3 vertexTangent = IN.m_tangent;
+    float3 vertexBitangent = cross(vertexNormal, vertexTangent);
+    if(ShouldHandleParallaxInDepthShaders())
+    {
+        float4x4 objectToWorld = GetObjectToWorld();
+        float3x3 objectToWorldIT = GetNormalToWorld();
+
+        ConstructTBN(IN.m_normal, IN.m_tangent, objectToWorld, objectToWorldIT, vertexNormal, vertexTangent, vertexBitangent);
```

Comment:
We could move this code down to the ShouldHandleParallaxInDepthShaders() section below, just to tidy things up a bit.